arm/Microsoft.RecoveryServices/vaults/deploy.bicep

```diff
@@ -126,43 +126,34 @@ resource rsv 'Microsoft.RecoveryServices/vaults@2021-08-01' = {
   properties: {}
 }
 
-module rsv_backupStorageConfig 'backupStorageConfig/deploy.bicep' = {
+module backupStorageConfiguration 'backupStorageConfig/deploy.bicep' = {
   name: '${uniqueString(deployment().name, location)}-BackupStorageConfig'
   params: {
     recoveryVaultName: rsv.name
-    vaultStorageType: backupStorageConfig.vaultStorageType
-    enableCRR: backupStorageConfig.enableCRR
+    storageModelType: backupStorageConfig.storageModelType
+    crossRegionRestoreFlag: backupStorageConfig.crossRegionRestoreFlag
   }
-  dependsOn: [
-    rsv
-  ]
 }
 
-module rsv_protectionContainers 'protectionContainers/deploy.bicep' = [for (protectionContainer, index) in protectionContainers: {
+module protectionContainer 'protectionContainers/deploy.bicep' = [for (protectionContainer, index) in protectionContainers: {
   name: '${uniqueString(deployment().name, location)}-ProtectionContainers-${index}'
   params: {
     recoveryVaultName: rsv.name
-    protectionContainerName: protectionContainer.protectionContainerName
+    name: protectionContainer.name
     sourceResourceId: protectionContainer.sourceResourceId
     friendlyName: protectionContainer.friendlyName
     backupManagementType: protectionContainer.backupManagementType
     containerType: protectionContainer.containerType
   }
-  dependsOn: [
-    rsv
-  ]
 }]
 
-module rsv_backupPolicies 'backupPolicies/deploy.bicep' = [for (backupPolicy, index) in backupPolicies: {
+module backupPolicy 'backupPolicies/deploy.bicep' = [for (backupPolicy, index) in backupPolicies: {
```

Comment:
Please change back to `rsv_backupPolicies`

Reply:
Done!